src/test/java/calculatortest/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,52 @@
+package calculatortest;
+
+import calculator.StringAddCalculator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringAddCalculatorTest {
+
+//    @ParameterizedTest
+    @ParameterizedTest
+    @NullAndEmptySource
+    void 빈문자열_혹은_null값이_들어오면_0(String value) {
+        assertThat(StringAddCalculator.splitAndSum(value)).isEqualTo(0);
+        assertThat(StringAddCalculator.splitAndSum(value)).isEqualTo(0);
+    }
```

Comment:
`value`에 `null`과 공백이 전달되어 각각 테스트가 실행이 되기 때문에, 테스트 코드는 한 줄이면 충분할 것 같습니다.
```suggestion
        assertThat(StringAddCalculator.splitAndSum(value)).isEqualTo(0);
    }
```